index.html

```diff
@@ -0,0 +1,39 @@
+<!DOCTYPE html>
+<html lang="ja">
+<head>
+    <meta charset="UTF-8">
+    <title>JavaScript Lesson1-4</title>
+</head>
+<body>
+    <script>
+      const tasks = {
+        '掃除' : '机を片付ける',
+        '買い物' : '牛乳を買う',
+        '運動' : '散歩する'
+      }
```

Comment:
オブジェクトの配列の方がいいと思います。

```suggestion
  const tasks = [
    {genre: '掃除', content: '机を片付ける'},
    {genre: '買い物', content: '牛乳を買う'},
    {genre: '運動', content: '散歩する'}
  ]
```